src/index.js

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
+      <App />
```

Comment:
## [OPTIONAL]
* This is an optional suggestion as it is not a requirement for this milestone but i strongly suggest you address it 🙏 
* Your calculator app seems not to be functioning i get an error whenever i click on any button maybe while working on this milestone you broke the code kindly make the necessary adjustments for this project to be fully functional.
* --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/52497246/134759094-ed528955-acb1-4ddc-831d-7c961ad92b02.png)